azure-pipelines.yaml

```diff
@@ -191,6 +190,11 @@ variables:
   - template: vars/input-variables.yaml@cnp-azuredevops-libraries
   - name: timeoutInMinutes
     value: 60
+  - name: finalAction
```

Comment:
why not just parameters.action?

Reply:
so parameters.action  will always print  `plan` as its default value even if its `apply`